### DISCUSSION TITLE
Is there a way to modify FEMA supply drop contents?

### DISCUSSION THREAD

**Discussion by Frostinice** [TEHE-Studios/ExpandedHelicopterEvents#166] [Q&A] [answered]
I was wondering if I could change its contents to something more for military supply drop (ammo, guns, rations, etc.). Otherwise, I could just rely on the ole admin powers.

**Comment by sharkster91** [comment, marked as answer]
Yes, but you would need to manually modify the loot files themselves. You
should probably stick to manually spawning them.
